viewing_party/party.py

```diff
@@ -1,23 +1,119 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    inputs = [title, genre, rating]
+    for input in inputs:

```

Comment:
This is a very robust way to implement a guard clause pattern! It could be simplified to refactor that loop like so:
```python
if not title or not genre or not rating:
    return None
```